src/App.tsx

```diff
@@ -60,33 +60,37 @@ export default class App extends React.Component<Props, State> {
                 <Layout.Sider className="Sider">
                     <Menu className="MenuItem" selectable={false} triggerSubMenuAction="click">
                         <a href="https://www.reduct.store" title="https://www.reduct.store">
-                            <Image src={logo} preview={false}/>
+                            <Image src={logo} preview={false} />
                         </a>
 
-                        <Divider/>
+                        <Divider />
                         {permissions ? <>
-                                <Menu.Item icon={<BorderOuterOutlined/>} onClick={() => history.push("/dashboard")}>
-                                    Dashboard
+                            <Menu.Item icon={<BorderOuterOutlined />} onClick={() => history.push("/dashboard")}>
+                                Dashboard
+                            </Menu.Item>
+                            <Menu.Item id="Buckets" icon={<DatabaseOutlined />}
+                                onClick={() => history.push("/buckets")}>
+                                Buckets
+                            </Menu.Item>
+                            <Menu.Item id="Replications" icon={<DatabaseOutlined />}
```

Comment:
1. Replications are only available for the full access
2. Let's find another icone different from the Buckets item